### DISCUSSION TITLE
markdown code block 支援語言設定

### DISCUSSION THREAD

**Discussion by toddLiao469469** [kwchang0831/svelte-QWER#51] [Ideas]
目前相關的設定是寫在 `QWER/mdify/renderer/default.js`裡的 `languages`
不知道是否能改成從 `UserConfig` 來設定呢？
但感覺可能會需要把prismjs的components改用dynamic import？

**Comment by kwchang0831** [comment]
我之前的想法是這樣的，這語法高亮的語言通常不會經常性的更動，所以我也就沒有特別花時間去改成動態import的方式，需要的直接加入就解決了。
但，若是後續要套用更新，的確是會被覆蓋掉。所以，我能理解改放到 `UserConfig` 的好處。
我當時也有在尋找其他的語法高亮的套件, 之後我的時間比較少就這樣放著了。

若您已經有solution了，歡迎來PR。
